code/modules/mob/living/living.dm

```diff
@@ -1050,6 +1050,7 @@
 	changeNext_move(CLICK_CD_RESIST)
 
 	SEND_SIGNAL(src, COMSIG_LIVING_RESIST, src)
+	cancel_camera()
```

Comment:
Make this check if a camera is actually being used before calling. This calls `unset_machine` unconditionally which could have some adverse side effects, either now or in the future.

Reply:
how do you check that? i already tried asking in coderbus 3 times and got ignored

Reply:
My first guess would be to see if `client.perspective` is ever set outside of a camera or AI.

Reply:
um..how would i actually check tho

Reply:
View variables yourself, go to client, see if its set

Go to a camera, see if its set then